main.go

```diff
@@ -73,85 +73,94 @@ func main() {
 		klog.Fatalf("Error building submariner clientset: %s", err.Error())
 	}
 
-	kubeInformerFactory := kubeInformers.NewSharedInformerFactoryWithOptions(kubeClient, time.Second*30, kubeInformers.WithNamespace(ss.Namespace))
-	submarinerInformerFactory := submarinerInformers.NewSharedInformerFactoryWithOptions(submarinerClient, time.Second*30, submarinerInformers.WithNamespace(ss.Namespace))
-	
+	kubeInformerFactory := kubeInformers.NewSharedInformerFactoryWithOptions(kubeClient, time.Second*30,
+		kubeInformers.WithNamespace(submSpec.Namespace))
+	submarinerInformerFactory := submarinerInformers.NewSharedInformerFactoryWithOptions(submarinerClient, time.Second*30,
+		submarinerInformers.WithNamespace(submSpec.Namespace))
+
 	start := func(context.Context) {
-		var ce cableengine.CableEngine
-		localCluster, err := util.GetLocalCluster(ss)
+		localCluster, err := util.GetLocalCluster(submSpec)
 		if err != nil {
-			klog.Fatalf("Fatal error occurred while retrieving local cluster %v", localCluster)
+			klog.Fatalf("Fatal error occurred while retrieving local cluster %v", err)
```

Comment:
Wouldn’t it be better to log both `localCluster` and `err`? How about using [fields](https://github.com/sirupsen/logrus#fields)?

Reply:
I'd have a more broader discussion to use logrus or any other logging library (which of course may make sense IMO) with everybody before pulling that dep :)

Regardless of that, I'd log both :)

Reply:
> I'd have a more broader discussion to use logrus or any other logging library (which of course may make sense IMO) with everybody before pulling that dep :)

Oh dear, I’m so used to having `logrus` that I didn’t check what Submariner is using...

Reply:
Hehehe :-D I suspected, yes

Reply:
If err is non-nil then localCluster is invalid, ie probably null

Reply:
Oh, yes, duh; `submSpec` then...